### ISSUE TITLE
[Catalog] The home screen tiles should be marked as buttons for accessibility

### ISSUE THREAD

**Issue by jverkoey** [material-components/material-components-ios#3877]
## Definition of done

- [ ] Each tile is marked as an accessibility button
- [ ] Each tile's a11y label is the tile title
